_posts/2017-03-14-functional-reactive-domain-modeling-book.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: "Book Review: Functional and Reactive Domain Modelling"
+author: ragb
+---
+
+    The Book [functional and reactive domain Modelling][book] ([Debasish Ghosh][debasish]) is one of the most interesting books I have read lately, in the land of Scala. This is not another beginner Scala book, neither a functional programming primmer. It is a much needed toor on model domain with functional patterns and concepts, reactive systems and modern software development in Scala. If you want to know how in the world one can use akka streams, free monads and actors interacting with each other, this book is a great start point :).
```

Comment:
land of Scala -> realm of Scala?
primmer -> primer?
toor -> tour
model domain -> domain modelling
akka -> Akka
start point -> starting point